src/werkzeug/datastructures.py

```diff
@@ -1264,6 +1264,17 @@ def __setitem__(self, key, value):
         else:
             self.set(key, value)
 
+    def update(self, other=(), **kwargs):
```

Comment:
Until we can use positional only args, need to use a name like "__other" since "other" could reasonably be used as a keyword arg.

Reply:
Or use `*args` and ensure that at most one arg is given.